core/src/main/resources/db/sql_migration_scripts/1.0.0.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE sample_list LIKE patient_list;
+INSERT sample_list SELECT * FROM patient_list;
+CREATE TABLE sample_list_list LIKE patient_list_list;
+INSERT sample_list_list SELECT * FROM patient_list_list;
+DROP TABLE micro_rna;
+DROP TABLE micro_rna_alteration;
+DROP TABLE mutation_frequency;
+ALTER TABLE sample_list_list CHANGE PATIENT_ID SAMPLE_ID INT(11);
+CREATE TABLE version (VERSION_NUMBER VARCHAR(8));
+INSERT into version values ("1.0.0");
```

Comment:
Can we put all migration sql into one script, e.g. migration.sql? We can put versions inside the sql, e.g.

```sql
##version: 1.0.0
CREATE TABLE info (DB_SCHEMA_VERSION VARCHAR(8));
INSERT into version values ("1.0.0");

##version: 1.1.0
CREATE TABLE sample_list LIKE patient_list;
INSERT sample_list SELECT * FROM patient_list;
CREATE TABLE sample_list_list LIKE patient_list_list;
INSERT sample_list_list SELECT * FROM patient_list_list;
DROP TABLE micro_rna;
DROP TABLE micro_rna_alteration;
DROP TABLE mutation_frequency;
ALTER TABLE sample_list_list CHANGE PATIENT_ID SAMPLE_ID INT(11);
```

The python code has to be modified.